config/x86/kernel-config

```diff
@@ -423,33 +419,34 @@ CONFIG_X86_DIRECT_GBPAGES=y
 CONFIG_NUMA=y
 CONFIG_AMD_NUMA=y
 CONFIG_X86_64_ACPI_NUMA=y
-# CONFIG_NUMA_EMU is not set
+CONFIG_NUMA_EMU=y
```

Comment:
Thinking about based on https://cateee.net/lkddb/web-lkddb/NUMA_EMU.html that does this actually make sense?